decidim/decidim
# Replace `bootstrap-tagsinput` npm package with `tom-select`

#### :tophat: What? Why?

We're using an unmaintained JS package for the input tags feature in /system panel, in the File uploads' organization form. 

Last release was 8 years ago: https://www.npmjs.com/package/bootstrap-tagsinput
This has a dependabot alert:

![Dependabot alerts for bootstrap-tagsinput](https://github.com/decidim/decidim/assets/717367/c6aad354-1a70-41b5-a7e5-1361033dad3a)

Sometime ago we discussed some alternatives for this feature (see #8683). One of the alternatives proposed was tom-select, that I think that we could also use for other select fields (as I mentioned in https://github.com/decidim/decidim/pull/11039#issuecomment-1600207306) 


#### :pushpin: Related Issues
 
- Related to #8683

#### Testing

1. Go to http://localhost:3000/system
2. Sign in as system@example.org
3. Edit the organization
4. Click on "Show advanced settings" 
5. Go to the "File uploads" section
6. It should work the same

### :camera: Screenshots
#### Before


https://github.com/decidim/decidim/assets/717367/fe7b8952-6e36-4e45-a40e-775652f099b6



#### After


https://github.com/decidim/decidim/assets/717367/cf9b9288-c5b5-43c8-942a-8d4a38342c68


:hearts: Thank you!
